WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,17 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ @BottomSeparator@ @InactiveCursor@ @FontSize-big@ padding:@SkinWrapperPadding@; padding-right:0px; padding-left:6px;
```

Comment:
The contained property 'Text-hint' property is defined as color:@lighten(TxtC,50)@, whcih has value of #999999 as in mock up